### DISCUSSION TITLE
Question about system of ODE and PDE

### DISCUSSION THREAD

**Discussion by changuniuni** [lululxvi/deepxde#1716] [Q&A]
Hi,
is it possible to solve system time-dependent PDE and ODE using deepXDE?

For example, the equation I am currently trying to solve is shown below.
![Screenshot 2024-04-18 at 3 40 25 PM](https://github.com/lululxvi/deepxde/assets/48553632/c09a6e7d-30ee-45a9-83c0-3339d0ce3af3)
Not only are ODEs and PDEs intertwined, but I need to integrate the function B with respect to "a" each time.
How should I use deepXDE in this case??

Thank you for your help :)

**Comment by lululxvi** [comment]
Implement numerical integration
